src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +52,49 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
+        MovieSession todayMovieSession = new MovieSession();
+        todayMovieSession.setCinemaHall(firstCinemaHall);
+        todayMovieSession.setMovie(fastAndFurious);
+        todayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector.getInstance(
+                MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
+        movieSessionService.add(todayMovieSession);
 
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
+        System.out.println(movieSessionService.get(todayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService) injector.getInstance(
+                AuthenticationService.class);
+        User loginBob;
```

Comment:
The variable `loginBob` is declared but not initialized before the try-catch block. If an exception occurs during registration or login, `loginBob` will remain uninitialized, leading to a potential `NullPointerException` when it is used later. Consider initializing `loginBob` with a default value or handling the case where registration or login fails.